docker/viame_girder_worker_gpu.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM kitware/viame:girder-worker-gpu-latest
```

Comment:
@Cookt2 Is this supposed to be built from the other included Dockerfile, `docker/girder_worker_gpu.docker`? This works correctly if I instead specify that image here, but leaving this as is and pulling down the image doesn't contain everything needed.

Reply:
@AlmightyYakob I imagine the published image just hasn't been updated since this is a PR and there's also no CI.